src/app/nav/navbar.component.css

```diff
@@ -1,10 +1,237 @@
+/* navbar styling */
+.navbar {
+  display: flex;
+  flex-direction: row;
+  align-items: center;
+  justify-content: flex-start;
+  padding: 10px 50px;
+  background-color: white;
+  color: rgb(31, 31, 31);
+  height: 90px;
+  font-size: large;
```

Comment:
these days you'll want to stick with an actual number. There are a crazy amount of ways you can reference a size. You should pick what makes sense to you after you review them, like `rem`, `em`, `px`, `pt`. These days I mostly see px. In the not so distant past it was all em.